tools/README.md

```diff
@@ -38,3 +38,14 @@ If you want to publish a package with specific version:
 ```
 ./publish-packages.sh --version="1.2.3" --scope="expo-permissions"
 ```
+
+## Versioning Android
```

Comment:
could you make sure that the release steps in universe/internal-docs are up-to-date as well?